e2e/terraform/terraform.tfvars

```
@@ -2,4 +2,4 @@ region               = "us-east-1"
 instance_type        = "t2.medium"
 server_count         = "3"
 client_count         = "4"
-windows_client_count = "1"
+windows_client_count = "0"
```

Incidental change - revert?

```suggestion
windows_client_count = "1"
```

---

Yikes! Sorry about that. Thanks for catching my bad git hygiene.